core/src/main/java/org/sonarsource/sonarlint/core/serverapi/branches/ProjectBranchesApi.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.serverapi.branches;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonarsource.sonarlint.core.serverapi.ServerApiHelper;
+
+public class ProjectBranchesApi {
+  private static final Logger LOG = Loggers.get(ProjectBranchesApi.class);
+
+  private static final String LIST_ALL_PROJECT_BRANCHES_URL = "/api/project_branches/list";
```

Comment:
Nitpick: I think the leading `/` should be removed for consistency.

Reply:
I used QualityProfileApi, SettingsApi and HotspotApi. All of them have leading / in paths.